monkey/tests/unit_tests/infection_monkey/master/test_ip_scanner.py

```diff
@@ -54,9 +54,23 @@ def stop():
     return Event()
 
 
+class ThreadSafeMagicMock:
```

Comment:
Looks highly reusable, should go in `monkey/tests/common`?

Reply:
`monkey/tests/common` is for things that are reusable _for common_. But maybe `monkey_itests/utils.py`.